internal/models/factor.go

```diff
@@ -117,7 +117,8 @@ func ParseAuthenticationMethod(authMethod string) (AuthenticationMethod, error)
 }
 
 type Factor struct {
-	ID           uuid.UUID          `json:"id" db:"id"`
+	ID uuid.UUID `json:"id" db:"id"`
+	// TODO: Consider removing this nested user field. We don't use it.
```

Comment:
```suggestion
	// TODO (joel): Consider removing this nested user field in late August. We don't use it.
```

Reply:
Not doing it now as it probably requires a little bit more care / checks to ensure it's not in use